code/game/gamemodes/changeling/traitor_chan.dm

```diff
@@ -16,7 +16,7 @@
 
 
 /datum/game_mode/traitor/changeling/pre_setup()
-	if(config.protect_roles_from_antagonist)
+	if(GLOB.configuration.gamemode.prevent_mindshield_antags)
```

Comment:
thinking about this one and may be over thinking and may be out of scope. I am assuming this is still going to follow the same logic for when a mindshield is removed during a game mode. this is most like just an issue of var name partly changed(prevent mindshield antags vs adding to restricted roles array) but code logic did not.